doc/source/whatsnew/v0.24.0.rst

```diff
@@ -1731,6 +1731,7 @@ Other
 
 - Bug where C variables were declared with external linkage causing import errors if certain other C libraries were imported before Pandas. (:issue:`24113`)
 - Require at least 0.28.2 version of ``cython`` to support read-only memoryviews (:issue:`21688`)
+- Fixing a `DeprecationWarning`: invalid escape sequence in versioneer.py (:issue:`24636`)
```

Comment:
this is not needed

Reply:
Done, I was not sure at first and added it after :)